runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingWorld.java

```diff
@@ -292,7 +328,7 @@ public boolean isInBounds(WorldPoint loc)
 		this.tabs = Collections.unmodifiableMap(umtabs);
 	}
 
-	private void add(FarmingRegion r, int... extraRegions)
+	private void add(Multimap<Integer, FarmingRegion> regions, FarmingRegion r, int... extraRegions)
```

Comment:
I dislike this change, especially since this is private.

Reply:
pmd was upset with FarmingWorld.regions not being final once the getter was removed. What is the suggested solution?

Reply:
you can opt it out of pmd since its incorrect.